src/framework/components/rigid-body/system.js

```diff
@@ -227,6 +236,42 @@ class ContactResult {
     }
 }
 
+/**
+ * Object holding the result of a hit on an Entity.
+ */
+class HitResult {
```

Comment:
This class looks identical to `RaycastResult` but with a different name. I would consider renaming `RaycastResult` to `HitResult` and adding the following to deprecated.js:

```
export const RaycastResult = HitResult;
```

Reply:
It's done!